<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hóa Đơn Bán Hàng</string>
  </property>
  <property name="styleSheet">
   <string>background-color: #f0f0f0;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <!-- Nhãn và trường khách hàng -->
   <widget class="QLabel" name="lblKhachHang">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Khách hàng:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboKhachHang">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px;</string>
    </property>
   </widget>
   <!-- Nhãn và trường ngày lập -->
   <widget class="QLabel" name="lblNgayLap">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ngày lập:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateNgayLap">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px;</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <!-- Trường tìm kiếm sản phẩm -->
   <widget class="QLineEdit" name="txtTimKiemSP">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Tìm kiếm sản phẩm...</string>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px;</string>
    </property>
   </widget>
   <!-- Bảng danh sách sản phẩm -->
   <widget class="QTableWidget" name="tableSanPham">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>650</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc;</string>
    </property>
   </widget>
   <!-- Bộ lọc hóa đơn -->
   <widget class="QLabel" name="lblTuNgay">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>260</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Từ ngày:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateTuNgay">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px;</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblDenNgay">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>260</y>
      <width>60</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Đến ngày:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateDenNgay">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc; border-radius: 3px;</string>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <!-- Bảng chi tiết hóa đơn -->
   <widget class="QTableWidget" name="tableChiTietHD">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>650</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string>border: 1px solid #ccc;</string>
    </property>
   </widget>
   <!-- Nhãn tổng tiền -->
   <widget class="QLabel" name="lblTongTien">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>200</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tổng tiền: 0</string>
    </property>
    <property name="styleSheet">
     <string>font-weight: bold;</string>
    </property>
   </widget>
   <!-- Các nút chức năng -->
   <widget class="QPushButton" name="btnThemSP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Thêm sản phẩm</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #4CAF50; color: white; border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXoaSP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Xóa sản phẩm</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #f44336; color: white; border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLuuHD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Lưu hóa đơn</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #2196F3; color: white; border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnXemHD">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Xem hóa đơn</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #9C27B0; color: white; border-radius: 5px;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnThoat">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Thoát</string>
    </property>
    <property name="styleSheet">
     <string>background-color: #607D8B; color: white; border-radius: 5px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>